jerry-core/ecma/builtin-objects/ecma-builtin-function.c

```diff
@@ -160,7 +160,7 @@ ecma_builtin_function_dispatch_construct (const ecma_value_t *arguments_list_p,
   ECMA_STRING_TO_UTF8_STRING (function_body_str_p, function_body_buffer_p, function_body_buffer_size);
 
 #if ENABLED (JERRY_LINE_INFO)
-  JERRY_CONTEXT (resource_name) = ecma_make_magic_string_value (LIT_MAGIC_STRING__EMPTY);
```

Comment:
Is it intended that the `|| ENABLED (JERRY_ERROR_MESSAGES)` guard is missing from here?

Reply:
Nope, I forgot to add it. Thanks.